testing/src/cypress/e2e/02-institution/06-manage-users.cy.ts

```diff
@@ -1,6 +1,7 @@
 import DashboardInstitutionObject from "../../page-objects/Institution-objects/DashboardInstitutionObject";
 import ManageUsersObject from "../../page-objects/Institution-objects/ManageUsersObject";
 import InstitutionHelperActions from "../../custom-command/institution/common-helper-functions.cy";
+import { it } from "mocha";
```

Comment:
just curious, why was it imported?

Reply:
I think i's IDE issue, when the cypress libraries does not load this was auto imported. Fixed it now